content/en/real_user_monitoring/funnel_analysis/_index.md

```diff
@@ -0,0 +1,92 @@
+
+## Analyzing a funnel
+
+After building a funnel, to understand why you see the conversion rate you do, the Funnel Insights panel offers correlated data on performance and user behavior trends.
+
+For high level trends, you can see the end-to-end conversion rate for your entire workflow and also see individual steps to step conversion to dropoff rates. If you want to understand what it looks like for someone who converted versus someone who dropped off, you can watch a session replay for each case.
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-analyzing-funnel.jpg" alt="Use the Funnel Insights panel to review performance and user behavior trends" style="width:90%;" >}}
+
+The performance section allows you to understand if poor performance could have affected conversion. You can view a graph with a correlation between the load time of that page and the conversion rate and also see if any issues (detected by Error Tracking) occurred on that page.
+
+The user behavior section allows you to compare the average frustration count (from [frustration signals][3]) with the conversion rate, and even drill into the frustration signals detected from individual actions. Next to this section is a chart showing the conversion and dropoff rate for specific countries, allowing you to understand if geographic region plays a role in a user converting. 
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-user-behavior.jpg" alt="User behavior section within funnel analysis" style="width:90%;" >}}
+
+## Share a funnel
+
+Funnels can be shared with your teams on dashboards to analyze conversion alongside other telemetry metrics, or in a [Notebook][4] to be used for reporting.
+
+Note that you have a few options when it comes to sharing:
+
+- Share the entire visualization to notebooks and dashboards
+
+  {{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-share-entire-visualization.jpg" alt="Share the entire visualization by clicking Export" style="width:90%;" >}}
+
+- Share individual widgets
+
+  {{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-share-individual-widgets-1.mp4" video="true" width=90% >}}
+
+## Alerting on conversion
+
+Setting up alerts on conversion/dropoff rates allows you to be notified when conversion drops below a predefined threshold. You can export the conversion rate query to a monitor in a few ways:
+
+- From the visualization: This option takes the end-to-end workflow's query, allowing you to alert on the conversion rate for an entire workflow.
+
+  {{< img src="real_user_monitoring/funnel_analysis/funnel-insights-alert-on-conversion-rate.mp4" alt="Create an alert on the conversion rate for an entire workflow" video=true width=90% >}}
+
+- From the Funnel Insights panel: This allows you to take individual queries and alert on them. While this could be the overall workflow, you also have the option to alert on individual steps' dropoff and conversion.
```

Comment:
```suggestion
- From the Funnel Insights panel: This allows you to take individual queries and alert on them. While this could be the overall workflow, you also have the option to alert on individual steps' conversion and drop off rates.
```

we say "conversion and drop off" everywhere else